src/components/video-card/VideoCard.css

```diff
@@ -1,6 +1,15 @@
+.ellipsis {
+	display: -webkit-box;
+	-webkit-line-clamp: 2;
+	-webkit-box-orient: vertical;
+	overflow: hidden;
+}
 .card {
 	position: relative;
-	min-width: 30rem;
+}
+.card-img-container {
+	height: 300px;
+	object-fit: fill;
```

Comment:
object-fit: contain or scale-down is looking better.
fill is giving feeling that images are compressed.

Reply:
object-fit: contain making white background around the images.so I put fill.